docs/site/src/demos/badges/SimpleBadge.js

```diff
@@ -0,0 +1,39 @@
+// @flow weak
+
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import Badge from 'material-ui/Badge';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import IconButton from 'material-ui/IconButton';
+import NotificationIcon from 'material-ui/svg-icons/notification';
+
+const styleSheet = createStyleSheet('SimpleBadge', () => ({
+  badge: {
```

Comment:
hmmm @oliviertassinari , I didn't get that.
1.  I am not sure what `MUI_SHEET_ORDER` is for? You mean adding it in `docs/site/src/components/App`?
2. That style is not needed, it is just for the demo. I have updated the example code, have a look